### PR TITLE
Remove IP address control feature (#7300)

### LINKED ISSUE #7300: Remove IP address control feature
The ip address control feature (See https://opfab.github.io/documentation/current/deployment/#_ip_address_control)  is not used, we need to remove it.

### PR DESCRIPTION
**Pull request by vlo-rte**
Nothing in release notes.

### CODE REVIEW

File: services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityConfiguration.java [hunk 1]
```diff
@@ -114,8 +114,7 @@ public static void configureCommon(final HttpSecurity http) throws Exception {
 
 
     public static AuthorizationManager<RequestAuthorizationContext> authenticatedAndIpAllowed() {
```
Comment (quinarygio) [inline services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityConfiguration.java]: Remove "ipAllowed" from methos names
Not needed to call AuthorizationManagers.allOf(..)  when there is just one AuthorizationManager

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OpfabAuthorizationManager.java [hunk 2]
```diff
@@ -21,17 +21,15 @@ private OpfabAuthorizationManager() {
     }
 
     public static AuthorizationManager<RequestAuthorizationContext> authenticatedAndIpAllowed() {
-        return AuthorizationManagers.allOf(AuthenticatedAuthorizationManager.authenticated(),
-                            new OpfabIpAuthorizationManager());
+        return AuthorizationManagers.allOf(AuthenticatedAuthorizationManager.authenticated());
```
Comment (quinarygio) [inline tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OpfabAuthorizationManager.java]: Remove "ipAllowed" from methos names.
Not needed to call AuthorizationManagers.allOf(..) when there is just one AuthorizationManager

**Review by quinarygio — CHANGES_REQUESTED** [review]

File: services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityConfiguration.java (hunk 1 above)
Comment (vlo-rte) [reply services/users/src/main/java/org/opfab/users/configuration/oauth2/WebSecurityConfiguration.java]: Done.

File: tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OpfabAuthorizationManager.java (hunk 2 above)
Comment (vlo-rte) [reply tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OpfabAuthorizationManager.java]: Done.

**Review by quinarygio — APPROVED** [review]

**Pull request merged**